src/Accounts/Accounts/Context/GetAzureRMContext.cs

```diff
@@ -69,9 +69,9 @@ protected override IAzureContext DefaultContext
         [Parameter(Mandatory = false, ParameterSetName = ListAllParameterSet, HelpMessage = "Refresh contexts from token cache")]
         public SwitchParameter RefreshContextFromTokenCache { get; set; }
 
-        protected override void BeginProcessing()
+        protected override bool RequireDefaultContext()
```

Comment:
why we need RequireDefaultContext()

Reply:
We need to overwrite this virtual function so that context check can be skipped.